storage_s3/indico_storage_s3/plugin.py

```diff
@@ -97,8 +97,11 @@ def getsize(self, file_id):
     def send_file(self, file_id, content_type, filename, inline=True):
         try:
             url = self.client.generate_presigned_url('get_object',
-                                                     Params={'Bucket': self.bucket, 'Key': file_id},
-                                                     ExpiresIn=100)
+                                                     Params={'Bucket': self.bucket,
+                                                             'Key': file_id,
+                                                             'ResponseContentDisposition': 'attachment',
```

Comment:
filename missing ;) also, inline/attachment should depend upon the `inline` argument of this method